src/controller.c

```diff
@@ -242,15 +242,54 @@ void run_controller()
   else// PASSTHROUGH
     _command.z = _combined_control.z.value;
 
+
+  // Add feedforward torques
+  _command.x += get_param_float(PARAM_X_EQ_TORQUE);
+  _command.y += get_param_float(PARAM_Y_EQ_TORQUE);
+  _command.z += get_param_float(PARAM_Z_EQ_TORQUE);
+
+
   _command.F = _combined_control.F.value;
 
   static uint32_t counter = 0;
   if (counter > 100)
   {
     mavlink_send_named_command_struct("RC", _rc_control);
     mavlink_send_named_command_struct("offboard", _offboard_control);
+    _combined_control.x.value += get_param_float(PARAM_X_EQ_TORQUE);
+    _combined_control.y.value += get_param_float(PARAM_Y_EQ_TORQUE);
+    _combined_control.z.value += get_param_float(PARAM_Z_EQ_TORQUE);
     mavlink_send_named_command_struct("combined", _combined_control);
     counter = 0;
   }
   counter++;
 }
+
+void calculate_equilbrium_torque_from_rc()
+{
+  // Make sure we are disarmed
+  if (!(_armed_state & ARMED))
+  {
+    // Tell the user that we are doing a equilibrium torque calibration
+    mavlink_log_warning("Capturing equilbrium offsets from RC");
+
+    // pass the rc_control through the controller
+    _combined_control.x = _rc_control.x;
+    _combined_control.y = _rc_control.y;
+    _combined_control.z = _rc_control.z;
+
+    run_controller();
+
+    // the output from the controller is going to be the static offsets
+    set_param_float(PARAM_X_EQ_TORQUE, _command.x);
+    set_param_float(PARAM_Y_EQ_TORQUE, _command.y);
+    set_param_float(PARAM_Z_EQ_TORQUE, _command.z);
+
+    mavlink_log_warning("Equilibrium torques found and applied.");
+    mavlink_log_warning("Please zero out trims on your transmitter");
+  }
+  else
+  {
+    mavlink_log_warning("Cannot perform equilbirum offset calibration while armed");
+  }
+}
```

Comment:
This is pretty cool, it should work even if RC is on angle mode and onboard is in rate mode or vice versa, right? That's pretty slick. A couple of potential problems I see:
 - Probably need to reset those torque params to zero before doing this calibration, since otherwise the old values would get applied and mess up your new values
 - This assumes that you land on a level surface. If you were flying in the park and landed at an angle in the grass, then ran this command, it would mess up your trim values (if you were tilted in the direction of your trim, the error between desired and actual would be zero so your control will be zero, while if you're tilted the opposite way, the computed torque would be extra big). Could we maybe temporarily overwrite `_current_state` to zero for this step to get around that or something?